taskcluster/android_taskgraph/parameters.py

```diff
@@ -0,0 +1,22 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from taskgraph.parameters import extend_parameters_schema
+from voluptuous import All, Any, Range, Required
+
+
+def get_defaults(repo_root):
+    return {
+        "pull_request_number": None,
+        "release_type": "",
+    }
```

Comment:
Same rationale here, we may want to remove this function: 

https://github.com/mozilla-mobile/firefox-android/blob/191d5fd17d0a9d610770db7668ea6563d715effa/taskcluster/android_taskgraph/__init__.py#L38-L43